BrokenTC2/TC/GearHandler.cpp

```diff
@@ -168,4 +169,27 @@ void GearHandler::gearDown() {
     switchSeqGear(false);
 }
 
+void GearHandler::holdFirstGear() {
+  if (m_settings.holdFirstGearWithClutch) {
+    windows::sendKeyboardEvent(m_settings.clutch,true);
+    std::this_thread::sleep_for(std::chrono::milliseconds(m_settings.keyDownTime));
+    windows::sendKeyboardEvent(m_settings.g1,true);
+  } else {
+    windows::sendKeyboardEvent(m_settings.g1,true);
+  }
+
+  m_currentGear = static_cast<Gear>(1);
+  emit gearChanged(toInt(m_currentGear));
+}
+
+void GearHandler::releaseFirstGear() {
+    if (m_settings.holdFirstGearWithClutch) {
+      windows::sendKeyboardEvent(m_settings.g1,false);

```

Comment:
Same goes for windows::sendKeyboardEvent(m_settings.g1,false);
It can be put outside of the condition ;)